### PR TITLE
Add UBERON:8410024 intestinal junction

### PR DESCRIPTION
**Pull request by paolaroncaglia**
Addresses #1722

### CODE REVIEW

**Comment by paolaroncaglia** [comment]
@dosumis 
@cmungall 
Following up on the discussion in #1722 , here's a basic solution. It hopefully addresses the needs of the Single Cell Expression Atlas, without delving too deep into the anatomical boundary/intestinal subdivision discussion. The reasoner infers 5 terms to be children of 'intestinal junction' and they look correct to me. The ones that are left out are cases where one of the two parts is not strictly intestine, e.g. rectosigmoid junction. If we wanted those (and other existing classes) to be included, we'd have to create a broader term including other parts of the digestive tract. A few more comments:
I couldn't find good dbxrefs or xrefs, so left David's ORCID only;
Similarly, I couldn't find other wordings/labels; we might ask Nancy George;
Looking at the asserted children of 'anatomical junction', I wonder if 'perineal body' is indeed a junction (def says it's located at a junction, which doesn't necessarily make it a junction?); should 'angle of oral opening' be better axiomatised? ; 'esophagogastric junction' is axiomatised differently from its siblings; and there may be a few more classes needing work?
Thanks.

**Comment by dosumis** [comment]
Got a bit carried away patternising.  Here's the result

image

@cmungall - can you review?  I think good for now.  Worth rolling into Doc (as DOSDP?) so we can standarise modelling choices.  I'm using connects through out, but in cleaning up I've see 'continuous with', overlaps and 'connected to' all used.  Probably matters less which choice we make than that we standardise.

File: uberon_edit.obo [hunk 1]
```diff
@@ -191394,6 +191396,17 @@ xref: FMA:16029
 intersection_of: UBERON:0000353 ! parenchyma
 intersection_of: part_of UBERON:0002106 ! spleen
 
+[Term]
+id: UBERON:1100000
+name: digestive tract junction
+def: "An anatomical junction between two parts of the digestive tract." [ORCID:0000-0002-7073-9172]
+synonym: "junction of intestine" EXACT []
```
Comment (mkeays) [inline uberon_edit.obo:191403]: not sure if it makes sense to have this as a synonym -- doesn't "digestive tract" include non-intestine parts as well (e.g. https://www.cancer.gov/publications/dictionaries/cancer-terms/def/digestive-tract )

Comment (paolaroncaglia) [reply uberon_edit.obo:191403]: @mkeays 
@dosumis 
"junction of intestine" should be a narrow synonym, not an exact one please.

Comment (paolaroncaglia) [reply uberon_edit.obo:191403]: (And you may want to add "intestinal junction" also as narrow synonym if not there already)

Comment (dosumis) [reply uberon_edit.obo:191403]: I'll remove.  I think it's there by accident because I clone a term to make this one.

**Review by dosumis — APPROVED** [review]

**Pull request merged**